measurements/api/measurements.py

```diff
@@ -147,8 +148,40 @@ def list_files(
         'results': results
     })
 
+
+
+def get_one_fastpath_measurement(measurement_id, download):
+    """Get one measurement from the fastpath table by measurement_id,
+    fetching the file from the fastpath host
+    """
+    log = current_app.logger
+    tid = measurement_id[len(FASTPATH_MSM_ID_PREFIX):]
+
+    conn = http.client.HTTPConnection("fastpath.ooni.nu", port=8000)
+    conn.request("GET", "/{}.json.lz4".format(tid))
+    r = conn.getresponse()
+    try:
+        assert r.status == 200
+        blob = r.read()
+        conn.close()
+        blob = lz4framed.decompress(blob)
+        response = make_response(blob)
+        response.headers.set('Content-Type', 'application/json')
+        return response
+    except Exception:
+        raise BadRequest("No measurement found")
+
+
+
 def get_measurement(measurement_id, download=None):
+    """Get one measurement by measurement_id,
+    fetching the file from S3 or the fastpath host as needed
+    Returns only the measurement without extra data from the database
+    """
     # XXX this query is SUPER slow
```

Comment:
We should probably drop this comment as it's actually no longer true.

Reply:
The slowness? It seems to be using approx .5 seconds according to the /api/v1.measurements_api_get_measurement metric on Prometheus

Reply:
I believe in fact it used to be even slower, but maybe I am mistaken.

In any case perhaps it's useful to add some extra details about the slowness (and maybe link to a github issue) also in light of the fact that the comment is now above a different bit of code than it was originally.

Reply:
Updated.